pcs/constraint.py

```diff
@@ -594,11 +593,10 @@ def normalize(constraint_el):
 
 
 def location_show(lib, argv, modifiers):
-    deprecation_warning(
-        "This command is deprecated and will be removed. "
-        "Please use 'pcs constraint location config' instead."
+    del lib, argv, modifiers
+    raise_command_replaced(
+        ["pcs constraint location config"], pcs_version="0.11"
     )
-    return location_config_cmd(lib, argv, modifiers)
```

Comment:
This is actually meant to be deprecated in pcs-0.11. Please, revert your changes.